addons/mail/__manifest__.py

```diff
@@ -138,6 +138,7 @@
             'mail/static/src/components/follower_subtype_list/follower_subtype_list.js',
             'mail/static/src/components/mail_template/mail_template.js',
             'mail/static/src/components/message/message.js',
+            'mail/static/src/components/message_actions/*',
```

Comment:
This is technically taking the xml too even though it is never used in that bundle.